### PR TITLE
test: Update topologies and test texts

### PR DESCRIPTION
**Pull request by mattiaswal**
All mgmt links should be lightgrey, all mgmt ports should be named mgmt links between duts should be named link.

## Checklist

Tick *relevant* boxes, this PR is-a or has-a:

- [ ] Bugfix
  - [ ] Regression tests
  - [ ] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

**Comment by troglobit** [comment]
Wow, that was quick! Nice work :100:

**Review by axkar — CHANGES_REQUESTED** [review]
Great work! A few comments only :)

**Review by jovatn — APPROVED** [review]
Great work!
See inline comments for proposed fixes - use what you think is useful.

A related note: OSPF multi-area still does not explain 11.0-network in topology or description, but mentioned in one of the test steps. Perhaps a small note in description, saying there are additional networks?

File: test/case/ietf_interfaces/bridge_vlan/topology.dot [hunk 1]
```diff
@@ -7,26 +7,26 @@ graph "2x2" {
...
 	host [
-	    label="host | { <mgmt1> mgmt1 | <data1> data1 | <mgmt2> mgmt2 }",
+	    label="host | { <mgmt1> mgmt1 | <data> data | <mgmt2> mgmt2 }",
 	    pos="0,16!",
 	    kind="controller",
 	];
 
         dut1 [
-	    label="{ <mgmt> mgmt | <data> data | <to_dut2> to_dut2} | dut1\n(10.0.0.2) ",
+	    label="{ <mgmt> mgmt | <data> data | <link> lnk2} | dut1\n(10.0.0.2) ",
```
[the remark this answers is not in the document]
Comment (mattiaswal) [reply test/case/ietf_interfaces/bridge_vlan/topology.dot:16]: Of course :+1:

File: test/case/ietf_interfaces/dual_bridge/Readme.adoc [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
 endif::testgroup[]
 ifndef::testgroup[]
-image::topology.png[Dual bridges topology]
+image::topology.png[Dual bridges on one device topology]
 endif::testgroup[]
 endif::topdoc[]
 ==== Test sequence
-. Initialize
+. Set up topology and attach to target DUTs
 . Configure two bridges linked with a veth pair furthest bridge has IP 10.0.0.2
```
[the remark this answers is not in the document]
Comment (mattiaswal) [reply test/case/ietf_interfaces/dual_bridge/Readme.adoc:25]: Redundant information, the IP is shown in ASCII art. Removing it

**Review by axkar — APPROVED** [review]
:1st_place_medal:

File: test/case/ietf_interfaces/bridge_vlan_separation/test.py [hunk 3]
```diff
@@ -1,39 +1,43 @@
...
-#        |                |     |                            |     |                 |
-#        |                |     |                            |     |                 |
-# ,-----------------------------------------------------------------------------------------,
-# |  host:mgmt0  host:data0     host:data1          host:data2     host:data3   host:mgmt1  |
-# |              [10.0.0.1]     [10.0.0.2]          [10.0.0.3]     [10.0.0.4]               |
-# |                (ns10)         (ns11)              (ns20)         (ns21)                 |
-# |                                                                                         |
-# |                                        [ HOST ]                                         |
-# '-----------------------------------------------------------------------------------------'
 """
```
[the remark this answers is not in the document]
Comment (mattiaswal) [reply test/case/ietf_interfaces/bridge_vlan_separation/test.py:2]: I agree :)

**Comment by mattiaswal** [comment]
> Great work! See inline comments for proposed fixes - use what you think is useful.
> 
> A related note: OSPF multi-area still does not explain 11.0-network in topology or description, but mentioned in one of the test steps. Perhaps a small note in description, saying there are additional networks?

Forgot that! There is  now an ASCII-art over loopback configuration.

**Pull request merged**